curp/src/server/raw_curp/log.rs

```diff
@@ -52,17 +54,17 @@ impl<C: 'static + Command> Log<C> {
             entries, // a fake log[0]
             commit_index: 0,
             last_applied: 0,
-            base_index: 1,
+            base_index: 0,
```

Comment:
Why don't we need fake log[0] entry now? Is there any use for it other than simplify the calculation?

Reply:
No, we still have a fake log[0]. I changed the meaning of base_index(see the comments). I think it's more straight forward this way.